lesson/outline/outline-template.md

```diff
@@ -1,104 +1,323 @@
-# Lesson Name
-
-Check out the [content standards](http://curriculum-documentation.codecademy.com/Content-Standards/outline/) for guidance on writing lesson outline.
+# Testing in Jest
 
 ## Lesson Information
 
 ### Resource(s)
 
-_Insert any resources you used while researching to help plan your lesson._
+- [jest official documentation](https://jestjs.io/)
+
 
 ### Learning Standards
-Check out the [content standards](http://curriculum-documentation.codecademy.com/Resources/outcomes-standards-objectives/#learning-standards) for guidance on writing learing standards.
 
-1. ...
-2. ...
-3. ...
+1. What is Jest
+2. Installing Jest
+3. Running Jest Test
+4. The `test()` function
+5. The `expect()` function and matchers
+6. False Positives in Jest with Asynchronous Code
+7. Testing Async Code - Async Callbacks
+8. Testing Async Code - Promises
+9. Mocking functions with `jest.fn()`
+10. Mocking modules with `jest.mock()`
+
 
 <hr>
 
-## Exercise # 1: _Insert exercise title here_
+## Exercise # 1: What is Jest
 
 ### Which course outcomes will be covered by this exercise?
 
-1. Learners will be able to...
+1. Learners will obtain a general understanding of testing and why it is important. 
+2. Learners will get brief exposure to the three main types of tests: 
+    - unit tests
+    - integration tests
+    - end-to-end tests. 
+3. Learners will understand the viability and efficiency of performing tests with jest. 
+
 
 ### Narrative Summary
 
-_Check out the [content standards](http://curriculum-documentation.codecademy.com/Content-Standards/narrative/) for guidance on writing narratives for exercises._
+1. In this lesson, we will explore the wonders of testing with jest and how it can expedite the project creation process. While there are many testing libraries out there, Jest provides a testing framework that focuses on simplicity, allowing us to do a lot, right out of the box without much setup.
+2. Will explain testing and talk about the three different types
+    - Unit tests 
+    - Integration tests 
+    - End-to-End tests
+3. Will define jest and some of its advantages 
+
+
+### What would you like to have in the workspace for this exercise? Share your plan below.
+
+#### Workplace Theme:
+- This lesson will be taught while testing out the functionalities of a single pre-developed application 
+#### Required Assets:
+- An infographic of unit test, integration tests, and end-to-end tests
+#### Components necessary:
+- I.E. text editor and output terminal
 
 
+<hr>
+
+## Exercise # 2: Setting up Jest
+
+### Which course outcomes will be covered by this exercise?
+
+1. Learners will be able to install Jest into a workplace and also configure jest with the ‘test’ script.
+
+
+### Narrative Summary
+
+1. Introduce the project that will be tested 
+2. Outline the steps to set up the workplace and  install jest
+3. Explain file labeling and structure to best use jest 
+4. Write test script
```

Comment:
I think I know what you're talking about here but for future outlines, it would be best to be a bit more explicit and include any specific details. For example, in addition to saying "Explain file labeling and structure...", you can briefly mention in your outline what those file labeling /structure best practices are (using **.test.js** or **.spec.js** suffixes or putting files in a **__test__** folder). 

Ditto for "Outline the steps to set up the workplace and install Jest". As a sub-bullet it would be nice to see what those steps are. The reason for this added granularity is that it helps the reviewer (me) get a better sense for the scope of the exercise. The more details you include, the better sense I have for knowing if the scope is appropriate or if its too long/too short.